packages/babel-preset-env/src/normalize-options.js

```diff
@@ -183,7 +186,8 @@ export function normalizeCoreJSOption(
 
   if (!useBuiltIns && version) {
     console.log(
-      "\nThe `corejs` option only has an effect when the `useBuiltIns` option is not `false`\n",
+      "\n[babel-preset-env]" +
+        "\nThe `corejs` option only has an effect when the `useBuiltIns` option is not `false`\n",
```

Comment:
And also here:

```suggestion
      "\nWARNING (@babel/preset-env): The `corejs` option only has an effect when the `useBuiltIns` option is not `false`\n",
```

(this is not technically a warning, but since it's probably an error in the user's config it's good to mark it as such).